pkg/cloud/services/ec2/launchtemplate.go

```diff
@@ -845,19 +873,24 @@ func (s *Service) buildLaunchTemplateTagSpecificationRequest(scope scope.LaunchT
 		Additional:  additionalTags,
 	})
 
-	if len(tags) > 0 {
-		// tag instances
+	// tag instances
+	{
```

Comment:
Do we use blocks like `{ }` on other parts of the code? I'm wondering if we want to use it, or if it improves things. If we don't use it anywhere else, I wouldn't use it.

Reply:
If there are zero tags per category (here: `ec2.ResourceTypeInstance` and `ec2.ResourceTypeVolume`), we just shouldn't add the tag specification. That's why the guard is there. And I think having a block, whether it's just `{ ... }` or `if ... { ... }` around each category, together with a comment above each block, separates the logic somewhat cleanly.